app/views/questions/show.html.erb

```diff
@@ -6,8 +6,9 @@
   <%= link_to "delete", question_path(@question), method: "delete" %><br>
 <% end %>
 
-
-
+<%= render partial: "answers/new", locals: {answer: @answer, question: @question}%>
+<h3>Answers</h3>
+<%= render partial: "answers/index", locals: {answers: @answers} %>
```

Comment:
If this partial were named properly to "answers" you could do this...

`render partial: "answers/answers", object: @answers`

and within the view use a magic method called `answers` to access the data.